src/Translator/assets/test/formatters/intl-formatter.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
For this file, I've reused some of the same test data from https://github.com/symfony/symfony/blob/015d5015e353ee5448bf7c350de0db4a03f8e13a/src/Symfony/Component/Translation/Tests/Formatter/IntlFormatterTest.php